packages/components/grid/src/grid.ts

```diff
@@ -457,7 +470,7 @@ export class Grid<T = any> extends ScopedElementsMixin(LitElement) {
       .filter(col => !col.hidden && col.autoWidth)
       .forEach(col => {
         const index = this.view.headerRows[this.view.headerRows.length - 1].indexOf(col),
-          cells = this.renderRoot.querySelectorAll<HTMLElement>(`:where(td, th):nth-child(${index + 1})`);
+          cells = this.renderRoot.querySelectorAll<HTMLElement>(`:where(tbody td, thead th):nth-child(${index + 1})`);
```

Comment:
what about th cells in the tbody, when you have a "heading" in the first column in the body for example.

Reply:
Is that correct HTML? I thought `<th>` elements are only allowed in the `<thead>` element?

Reply:
Changed